python/cudf/cudf/core/column/datetime.py

```diff
@@ -726,9 +726,9 @@ def to_pandas(
     ) -> pd.Series:
         if nullable:
             raise NotImplementedError(f"{nullable=} is not implemented.")
-        series = self._local_time.to_pandas().dt.tz_localize(
-            self.dtype.tz, ambiguous="NaT", nonexistent="NaT"
-        )
```

Comment:
Are there any tests we can write to cover these cases?

Reply:
I'll be reverting this change anyways. I'm encountering a possible pandas bug with less common timezones where roundtripping through UTC near an ambiguous time does not return the same local time